code/search/variants/SearchVariant.php

```diff
@@ -142,7 +152,7 @@ public static function with($class = null, $includeSubclasses = true)
      *
      * SearchVariant::call(...) ==== SearchVariant::with()->call(...);
      */
-    public static function call($method, &$a1=null, &$a2=null, &$a3=null, &$a4=null, &$a5=null, &$a6=null, &$a7=null)
+    public static function call($method, &$a1 = null, &$a2 = null, &$a3 = null, &$a4 = null, &$a5 = null, &$a6 = null, &$a7 = null)
```

Comment:
We can probably replace this with a variadic argument now that PHP 5.6 is the minimum